priam/src/main/java/com/netflix/priam/backup/SnapshotBackup.java

```
@@ -180,11 +190,50 @@ private void notifyObservers() {
+        abstractBackupPaths.addAll(upload(snapshotDir, BackupFileType.SNAP));
+    }
+
+    private void findForgottenFiles(File snapshotDir) {
+        try {
+            Collection<File> snapshotFiles = FileUtils.listFiles(snapshotDir, FileFilterUtils.fileFileFilter(), null);
+            File columnfamilyDir = snapshotDir.getParentFile().getParentFile();
+            Collection<File> columnfamilyFiles = FileUtils.listFiles(columnfamilyDir, FileFilterUtils.fileFileFilter(), null);
+
+            for (File file : snapshotFiles) {
+                //Get its parent directory file based on this file.
+                File originalFile = new File(columnfamilyDir, file.getName());
+                columnfamilyFiles.remove(originalFile);
+            }
+
+            if (columnfamilyFiles.size() == 0)
+                return;
+
+            //There might be new SSTables since we took snapshot. Remove them from this list.
+            Collection<File> forgottenFiles = columnfamilyFiles.stream().filter(file -> file.isFile()).filter(file -> {
+                Instant fileModification = Instant.ofEpochMilli(file.lastModified());
+                return fileModification.isBefore(snapshotInstant);
+            }).collect(Collectors.toList());
+
+            forgottenFiles.parallelStream().forEach(file -> logger.info("Forgotten file: {} for CF: {}", file.getAbsolutePath(), columnfamilyDir.getName()));
+
+            if (forgottenFiles == null || forgottenFiles.size() == 0)
+                return;
+
+            backupMetrics.incrementForgottenFiles(forgottenFiles.size());
+            logger.warn("Forgotten files: {} found for CF: {}", forgottenFiles.size(), columnfamilyDir.getName());
+        }catch (Exception e)
+        {
+            //Eat the exception, if there, for any reason. This should not stop the snapshot for any reason.
+            logger.error("Exception occurred while trying to find forgottenFile. Ignoring the error and continue with backup.", e);
```

nit: readability: "Ignoring the error and continuing with remaining backup."